src/app/modules/share/header/user-info.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from "@angular/common/http";
+import { environment } from "../../../../environments/environment";
```

Comment:
U can add short path for this case. Please see example https://medium.com/rpdstartup/simplifying-typescript-import-paths-in-angular-c3249a4edbb3

Reply:
Fixed